### ISSUE TITLE
Lnd panic error on node startup

### ISSUE THREAD

**Issue by adolfosrs** [lightningnetwork/lnd#3071]
### Background

My node is shutting down when I unlock it and I'm not being able to close my open channels.

Have made some research and thought it could be database corruption so I've restored my node to a previous state i had backed up but the problem is still happening.

### Your environment

* Version: 0.6.0-beta commit=v0.6-beta
* Linux adolfosrs 4.14.98-v7+ #1200 SMP Tue Feb 12 20:27:48 GMT 2019 armv7l GNU/Linux
* Bitcoin Core Daemon version v0.17.1
* Raspiblitz

### Steps to reproduce

Full logs: https://termbin.com/728t

```
May 12 12:01:35 adolfosrs systemd[1]: Starting LND Lightning Daemon...
May 12 12:01:35 adolfosrs systemd[1]: Started LND Lightning Daemon.
May 12 12:10:57 adolfosrs lnd[22410]: panic: runtime error: index out of range
May 12 12:10:57 adolfosrs lnd[22410]: goroutine 1 [running]:
May 12 12:10:57 adolfosrs lnd[22410]: github.com/coreos/bbolt.(*DB).page(...)
May 12 12:10:57 adolfosrs systemd[1]: lnd.service: Main process exited, code=exited, status=2/INVALIDARGUMENT
May 12 12:10:57 adolfosrs systemd[1]: lnd.service: Unit entered failed state.
May 12 12:10:57 adolfosrs systemd[1]: lnd.service: Failed with result 'exit-code'.
May 12 12:11:57 adolfosrs systemd[1]: lnd.service: Service hold-off time over, scheduling restart.
May 12 12:11:57 adolfosrs systemd[1]: Stopped LND Lightning Daemon.
May 12 12:11:57 adolfosrs systemd[1]: Starting LND Lightning Daemon...
May 12 12:11:57 adolfosrs systemd[1]: Started LND Lightning Daemon.
```

### Expected behaviour

Startup should run without errors and I should have access to my open channels

### Actual behaviour

Lnd node shuts down due to a panic error.

**Comment by wpaulino** [comment]
> so I've restored my node to a previous state i had backed up

This is not safe to do as you could potentially end up breaching yourself and lose your channel funds.

It does indeed look like database corruption. If you have your seed and a `channel.backup`, you can recover your off-chain funds as well as on-chain. Check out this [guide](https://github.com/lightningnetwork/lnd/blob/master/docs/recovery.md).

**Comment by adolfosrs** [comment]
@wpaulino Thanks for helping.

Unfortunately I was on version 0.5.1 when the corruption occurred so I don't have a `channel.backup`.

Backing up using a [lnd-rescue file](https://github.com/rootzoll/raspiblitz/blob/master/FAQ.md#how-can-i-recover-my-coins-from-a-failing-raspiblitz) ends up in the same error. So it makes me think that could be something else since my backup refers to a state that was not crashing.

**Comment by Roasbeef** [comment]
The issue with unclean shutdowns has been fixed in 0.6, but if this happened prior to that, then it's possible the on-disk state may be corrupted. You can recover your on-chain funds with your seed. It's possible to attempt to retroactively create SCBs, but that requires manual directed work for this specific case.

**Comment by adolfosrs** [comment]
> It's possible to attempt to retroactively create SCBs, but that requires manual directed work for this specific case.

@Roasbeef Is there any documentation you could point me regarding it? Thanks!
